doc/man3/EVP_MAC.pod

```diff
@@ -381,6 +385,9 @@ return 1 on success, 0 on error.
 EVP_MAC_CTX_get_mac_size() returns the expected output size, or 0 if it isn't set.
 If it isn't set, a call to EVP_MAC_init() should get it set.
 
+EVP_MAC_CTX_get_block_size() returns the block size, or 0 if it isn't set.
+If it isn't set, a call to EVP_MAC_init() should get it set.
```

Comment:
get it set?

Reply:
That's what was written for three lines up which I copied from :)